src/Laravel/Eloquent/Extension/FilterQueryExtension.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Laravel\Eloquent\Extension;
+
+use ApiPlatform\Laravel\Eloquent\Filter\FilterInterface;
+use ApiPlatform\Metadata\Operation;
+use ApiPlatform\State\ParameterNotFound;
+use Illuminate\Database\Eloquent\Builder;
+use Psr\Container\ContainerInterface;
+
+final readonly class FilterQueryExtension implements QueryExtensionInterface
+{
+    /**
+     * @param ContainerInterface<FilterInterface> $filterLocator
+     */
+    public function __construct(private ContainerInterface $filterLocator)
```

Comment:
```suggestion
    public function __construct(
    private readonly ContainerInterface $filterLocator,
)
```

Reply:
class is readonly, fixed the constructor on several lines